test/regression.uts

```diff
@@ -188,12 +188,23 @@ assert p == "127.0.0.1"
 
 = Interface related functions
 
+import mock
+
 conf.iface
 
 get_if_raw_hwaddr(conf.iface)
 
 bytes_hex(get_if_raw_addr(conf.iface))
 
+addr = u"lladdr 29:0b:c2:ff:fe:53:21:e9\n"
```

Comment:
You need to move this test below and make it not run on windows + rebase and the PR will be ready !